bpf/lib/csum.h

```diff
@@ -25,6 +25,7 @@
 
 #define TCP_CSUM_OFF (offsetof(struct tcphdr, check))
 #define UDP_CSUM_OFF (offsetof(struct udphdr, check))
+#define SCTP_CSUM_OFF 8
```

Comment:
nit: `(offsetof(struct sctphdr, checksum))`

(From `linux/sctp.h`)

Reply:
I looked in my `linux/sctp.h` and did not find that header. I assume it's only present in a later dev version. I'll get a copy from a recent kernel.